test-data/unit/cmdline.test

```diff
@@ -170,23 +170,34 @@ def f():
     except ZeroDivisionError, err:
         print err
 
-[case testNoConfigFile]
-# cmd: mypy main.py --config-file=
-[file mypy.ini]
+[case testWarnUnusedIgnores]
+# cmd: mypy main.py --config-file=pyproject.toml
+[file pyproject.toml]
 \[mypy]
-warn_unused_ignores = True
+warn_unused_ignores = true
+[file main.py]
+# type: ignore
+[out]
+main.py:1: error: unused 'type: ignore' comment
+
+[case testMissingMypySection]
```

Comment:
Hm, I think it may be best not to do this. It would be annoying to have mypy fail all of a sudden on projects with a pyproject.toml just because they don't have a mypy section.